cpp/CMakeLists.txt

```diff
@@ -3,11 +3,26 @@ cmake_minimum_required(VERSION 3.10)
 project(toppra)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
+# Dynamics
+option(BUILD_WITH_PINOCCHIO "Compile with Pinocchio library" OFF)
```

Comment:
Are there any instructions for building and installing pinocchio/qpoases?

Reply:
On my side, for both I use robotpkg : http://robotpkg.openrobots.org/debian.html

Then: sudo apt install robotpkg-qpoases robotpkg-py27-pinocchio (or robotpkg-py36-pinocchio or another python version).

Otherwise, you can install them from sources.

Reply:
For Pinocchio: https://stack-of-tasks.github.io/pinocchio/download.html

Reply:
For qpOASES, from source, I don't know. https://github.com/coin-or/qpOASES/blob/master/INSTALL gives some intructions but, as they mention svn, I guess they are not up-to-date.